### DISCUSSION TITLE
Persist login

### DISCUSSION THREAD

**Discussion by Destrocamil** [drakkan/sftpgo#1311] [Q&A]
Hey! 
Whenever I login, if I close the tab, I have to re-login again.
I am using this tool to access some S3 folders I have where I store some binaries I build using Gitlab. On Gitlab, I create a release and add release assets, mainly the link to my sftpgo instance -> folder where binaries are for that release version.
What happens is that, if I click the link, I have to login and I am not sent to the link I had, instead to the home folder. Only way is to, after login and inside the same tab, copy the URL again.
Is there a way to circumvent this?

**Comment by drakkan** [comment]
If you close the tab you should not have to login again, see here

[Video del 2023-05-30 17-42-52.webm](https://github.com/drakkan/sftpgo/assets/553263/30789892-e391-4935-ba6a-4e7bf2adf7d9)

Once the login session expires and you are redirected to the login screen the original url is lost. I think we could add a feature to preserve the url in the future but it isn't a priority for now, sorry
